manifests/init.pp

```diff
@@ -109,140 +105,186 @@
+      $default_sshd_config_mode                    = '0600'
+      $default_sshd_config_use_dns                 = 'yes'
+      $default_sshd_config_xauth_location          = '/usr/bin/xauth'
+      $default_sshd_use_pam                        = 'yes'
+      $default_sshd_gssapikeyexchange              = undef
+      $default_sshd_pamauthenticationviakbdint     = undef
+      $default_sshd_gssapicleanupcredentials       = 'yes'
+      $default_sshd_acceptenv                      = true
+      $default_service_hasstatus                   = true
+      $default_sshd_config_serverkeybits           = '1024'
+      $default_sshd_config_hostkey                 = [ '/etc/ssh/ssh_host_rsa_key' ]
+      $default_sshd_addressfamily                  = 'any'
       case $::architecture {
         'x86_64': {
           if ($::operatingsystem == 'SLES') and ($::operatingsystemrelease =~ /^12\./) {
-            $default_sshd_config_subsystem_sftp = '/usr/lib/ssh/sftp-server'
+            $default_sshd_config_subsystem_sftp    = '/usr/lib/ssh/sftp-server'
           } else {
-            $default_sshd_config_subsystem_sftp = '/usr/lib64/ssh/sftp-server'
+            $default_sshd_config_subsystem_sftp    = '/usr/lib64/ssh/sftp-server'
           }
         }
         'i386' : {
-          $default_sshd_config_subsystem_sftp = '/usr/lib/ssh/sftp-server'
+          $default_sshd_config_subsystem_sftp      = '/usr/lib/ssh/sftp-server'
       }
         default: {
           fail("ssh supports architectures x86_64 and i386 for Suse. Detected architecture is <${::architecture}>.")
         }
       }
     }
     'Debian': {
-      $default_packages                        = ['openssh-server',
-                                                  'openssh-client']
-      $default_service_name                    = 'ssh'
-      $default_ssh_config_forward_x11_trusted  = 'yes'
-      $default_ssh_config_hash_known_hosts     = 'no'
-      $default_ssh_package_source              = undef
-      $default_ssh_package_adminfile           = undef
-      $default_ssh_sendenv                     = true
-      $default_sshd_config_subsystem_sftp      = '/usr/lib/openssh/sftp-server'
-      $default_sshd_config_mode                = '0600'
-      $default_sshd_config_use_dns             = 'yes'
-      $default_sshd_config_xauth_location      = '/usr/bin/xauth'
-      $default_sshd_use_pam                    = 'yes'
-      $default_sshd_gssapikeyexchange          = undef
-      $default_sshd_pamauthenticationviakbdint = undef
-      $default_sshd_gssapicleanupcredentials   = 'yes'
-      $default_sshd_acceptenv                  = true
-      $default_service_hasstatus               = true
-      $default_sshd_config_serverkeybits       = '1024'
-      $default_sshd_config_hostkey             = [ '/etc/ssh/ssh_host_rsa_key' ]
-      $default_sshd_addressfamily              = 'any'
+      $default_packages                            = ['openssh-server',
```

Comment:
It is needed because $default_ssh_config_global_known_hosts_group is too long and shifted the border - so I leveled all the other params